.scripts/make-migration.ts

```diff
@@ -0,0 +1,50 @@
+import { readdirSync, readFileSync, writeFileSync } from 'fs';
+
+import { renderFile } from 'template-file';
+
+function main(number: string, comment: string): void {
```

Comment:
if I don't provide a comment it is throwing this error:
```
/Users/diegosampaio/dev/Rocket.Chat/.scripts/make-migration.ts:11
	if (comment.trim()) {
             ^
TypeError: Cannot read property 'trim' of undefined
    at main (/Users/diegosampaio/dev/Rocket.Chat/.scripts/make-migration.ts:11:14)
```

this might be a solution:
```suggestion
function main(number: string, comment = ''): void {
```

Reply:
Done!